src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs

```diff
@@ -14,7 +14,10 @@ public static class KeycloakResourceBuilderExtensions
 {
     private const string AdminEnvVarName = "KEYCLOAK_ADMIN";
     private const string AdminPasswordEnvVarName = "KEYCLOAK_ADMIN_PASSWORD";
+    private const string HealthCheckEnvVarName = "KC_HEALTH_ENABLED";
     private const int DefaultContainerPort = 8080;
+    private const int HealthCheckContainerPort = 9000;
```

Comment:
Seems Keycloak refers to this endpoint as the "management interface": https://www.keycloak.org/server/management-interface

Let's change the names here accordingly:

```suggestion
    private const string ManagementInterfaceContainerPort = 9000;
```